.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Release
+about: "*For maintainers only*"
+title: "release: 0.0.0"
+labels: ""
+assignees: "@cdr/code-server-reviewers"
+---
+
+<!-- Maintainer: fill out the checklist -->
+
+## Checklist
+
+- [ ] Assign to next release manager
```

Comment:
Since there is only three of us (currently), I think it's easy to know who the next release manager should be each time. But open to other ideas if we think this should be documented.